.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: ci
+
+on:
+  # Trigger the workflow on push to master or develop, except tag creation
+  push:
+    branches:
+      - 'master'
+      - 'develop'
```

Comment:
It should be main (also in the comment above)

Reply:
Good point.